src/elements/select/select.svelte

```diff
@@ -105,12 +106,14 @@ const handleEnter = () => {
 
     if (result) {
       value = result;
+      dispatch('change', { value });
+    } else {
+      value = '';
     }
   }
   if (open) {
     input.blur();
   }
-  dispatch('input', { value });
```

Comment:
Minor point, but should this `input` event remain? Experimenting with a vanilla `<select>`, selecting an option with [enter] fires both events

Reply:
there are likely v-selects in app that are listening to this as well